src/platform/echarts/base/base.types.ts

```diff
@@ -0,0 +1,38 @@
+type TdNameLocation = 'start' | 'middle' | 'center' | 'end';
+
+type TdAxisType = 'value' | 'category' | 'time' | 'log';
+
+type TdAxisLineType = 'solid' | 'dashed' | 'dotted';
+
+type TdPosition = 'left' | 'right';
```

Comment:
So this is not being used anywhere, but specifically about axis.. there are 2 types of positions:

1 for xAxis which is `top` and `bottom` and 1 for yAxis which is `left` and `right`.. so if we wanna create to enums and use them would be nice, if not its fine.

Reply:
Are you saying remove them and didn’t them in the c and y axis component respectively?

Reply:
Well, we either need to create 2 enums and use them respectively in:

https://github.com/Teradata/covalent-echarts/blob/develop/src/platform/echarts/base/axis/x-axis.component.ts#L17

and 

https://github.com/Teradata/covalent-echarts/blob/develop/src/platform/echarts/base/axis/y-axis.component.ts#L17

since as it is right now, the enum is not used.

Reply:
Alright probably makes sense to move these into axis.interface.ts if they are specific to the axis components. Sounds like you prefer Enums over literals is that correct? Which I guess helps if the API ever changes we would need to only change in one spot, any other benefits to enums? I’ll add a position for the y axis definitely missed that